updating/index.adoc

```diff
@@ -0,0 +1,67 @@
+:_content-type: ASSEMBLY
+[id="updating-clusters-overview"]
+= Updating clusters overview
+include::_attributes/common-attributes.adoc[]
+:context: updating-clusters-overview
+
+toc::[]
+
+You can update a {product-title} cluster with a single operation by using the web console or the OpenShift CLI (oc) with {product-title} 4.
+
+[id="updating-clusters-overview-understanding-container-platform-service"]
+== Understanding OpenShift Container Platform service
```

Comment:
If you do keep the title as it is, you could replace "OpenShift Container Platform" with "{product-title}".

Reply:
Ack